src/test/java/pages/WebDriverSingletion.java

```diff
@@ -0,0 +1,51 @@
+package pages;
+
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+public class WebDriverSingletion {
+
+    private static WebDriverSingletion instance = null;
+    private WebDriver driver;
+
+    private WebDriverSingletion() {
+    }
+
+    public static WebDriverSingletion getInstance() {
+        if (instance == null) {
+            instance = new WebDriverSingletion();
+        }
+        return instance;
+    }
+
+    public WebDriver openBrowser() {
```

Comment:
If you will open all page object classes, then you will see, that it looks like you're opening browser on each page) Please rename